Documentation/user-guides/getting-started.md

```diff
@@ -1,36 +1,72 @@
+This guide will show you how to deploy the Prometheus operator, set up a
+Prometheus instance, and configure metrics collection for a sample application.
 
-The Prometheus Operator serves to make running Prometheus on top of Kubernetes as easy as possible, while preserving Kubernetes-native configuration options.
+{{< alert icon="👉" text="Prometheus Operator requires use of Kubernetes v1.16.x and up."/>}}
 
-## Example Prometheus Operator manifest
+> Note: [Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+> were introduced by CoreOS as a class of software that operates other software,
+> putting operational knowledge collected by humans into software.
 
-To follow this getting started you will need a Kubernetes cluster you have access to. This [example](../../bundle.yaml) describes a Prometheus Operator Deployment, and its required ClusterRole, ClusterRoleBinding, Service Account and Custom Resource Definitions.
+## Pre-requisites
 
-## Related resources
+To follow this guide, you will need a Kubernetes cluster with admin permissions.
 
-The Prometheus Operator introduces additional resources in Kubernetes to declare the desired state of a Prometheus and Alertmanager cluster as well as the Prometheus configuration. The resources it introduces are:
+## Installing the operator
+
+The first step is to install the operator's Custom Resource Definitions (CRDs) as well
+as the operator itself with the required RBAC resources.
+
+Run the following commands to install the CRDs and deploy the operator in the `default` namespace:
+
+```bash
+LATEST=$(curl -s https://api.github.com/repos/prometheus-operator/prometheus-operator/releases/latest | jq -cr .tag_name)
+curl -sL https://github.com/prometheus-operator/prometheus-operator/releases/download/${LATEST}/bundle.yaml | kubectl create -f -
+```
+
+It can take a few minutes for the operator to be up and running. You can check for completion with the following command:
+
+```bash
+kubectl wait --for=condition=Ready pods -l  app.kubernetes.io/name=prometheus-operator -n default
+```
+
+The Prometheus Operator introduces custom resources in Kubernetes to declare
+the desired state of a Prometheus and Alertmanager cluster as well as the
+Prometheus configuration. For this guide, the resources of interest are:
 
 * `Prometheus`
-* `Alertmanager`
 * `ServiceMonitor`
+* `PodMonitor`
+
+The `Prometheus` resource declaratively describes the desired state of a
+Prometheus deployment, while `ServiceMonitor` and `PodMonitor` resources
+describe the targets to be monitored by Prometheus.
 
-> See the [Alerting guide](alerting.md) for more information about the `Alertmanager` resource, or the [Design document](../design.md) for an overview of all resources introduced by the Prometheus Operator.
+![Prometheus Operator Architecture](/img/architecture.png)
```

Comment:
Incorrect img path, it should be `Documentation/img/architecture.png`

Reply:
nope, this is required to display correctly here: https://prometheus-operator.dev/docs/user-guides/getting-started/#installing-the-operator

Reply:
Sorry, I was trying to access that image from the markdown and it was throwing an error - https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md